<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Top Notes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="noteTableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>671</width>
      <height>521</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="noteCreateButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>New Note</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noteBumpUpButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bump Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noteDeleteButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color: red;}</string>
    </property>
    <property name="text">
     <string>Delete Note</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="noteBrowser">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>210</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="detailBrowser">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>340</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="noteBumpDownButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>50</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Bump Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noteUpdateButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>130</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Note</string>
    </property>
   </widget>
   <widget class="QComboBox" name="categorySelect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>&lt;all&gt;</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="categoryField">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
